src/polymer/js-utils.ts

```diff
@@ -146,30 +146,26 @@ export function toScannedMethod(
 export function getOrInferPrivacy(
     name: string,
     annotation: jsdoc.Annotation|undefined,
-    privateUnlessDocumented: boolean): Privacy {
+    inferFromName = true,
```

Comment:
Well, how about 'abstract' classes? Its not hard to think of scenarios where you might want to implement a common base class for a group of elements that is not meat to be instantiated or used otherwise outside a given scope. Imo your argument for 'top-level members' applies to to classes as well:

> I think top-level members of a file should be assumed to not be visible unless they're part of a namespace, or they're defined elements. That's a good approximation of a scope for now.